source/ncengine/graphics2/frontend/subsystem/CameraSubsystem.cpp

```diff
@@ -43,13 +43,22 @@ auto CameraSubsystem::BuildState(ecs::ExplicitEcs<Transform> ecs) -> CameraRende
                 m_mainCamera->ViewMatrix(),
                 m_mainCamera->ProjectionMatrix()
             ),
-            .position = transform.Position()
+            .invProjection = m_mainCamera->InverseProjectionMatrix(),
+            .position = transform.Position(),
+            .nearClip = g_defaultProperties.nearClip,
```

Comment:
The camera has configurable near/far clip values that are probably needed here. I don't think there's a getter yet.

Reply:
Done